scripts/apps/monitoring/views/monitoring-view.html

```diff
@@ -141,6 +141,7 @@ <h2 class="subnav__page-title sd-flex-no-grow sd-empty" ng-hide="monitoring.sing
                                 class="navbtn navbtn--white"
                                 sd-tooltip="{{ 'Create new item' | translate }}"
                                 flow="left"
+                                data-test-id="create-new-item"
```

Comment:
`data-test-id` for this one already exists [content-create](https://github.com/superdesk/superdesk-client-core/blob/develop/scripts/core/ui/components/content-create-dropdown/content-create-dropdown.tsx#L22)